### ISSUE TITLE
How to use dash's pattern matching callback?

### ISSUE THREAD

**Issue by kino4u** [GibbsConsulting/django-plotly-dash#277]
I tried to replicate one of the simple examples of the [pattern matching callbacks](https://dash.plotly.com/pattern-matching-callbacks). None of them works with `django-plotly-dash`. I'm getting the following error:

```
Internal Server Error: /django/project/django_plotly_dash/app/PatternMatchAll/_dash-update-component
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.7/site-packages/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/usr/local/lib/python3.7/site-packages/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/usr/local/lib/python3.7/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/django_plotly_dash/views.py", line 74, in update
    return _update(request, ident, stateless, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/django_plotly_dash/views.py", line 93, in _update
    resp = view_func()
  File "/usr/local/lib/python3.7/site-packages/django_plotly_dash/dash_wrapper.py", line 560, in dispatch
    return self.dispatch_with_args(body, argMap=dict())
  File "/usr/local/lib/python3.7/site-packages/django_plotly_dash/dash_wrapper.py", line 622, in dispatch_with_args
    if c['property'] == component_registration['property'] and c['id'] == component_registration['id']:
TypeError: list indices must be integers or slices, not str
```

When I debugged the error, I noticed that `inputs` is a nested list. So I added the following `if` statements to get rid of the nested list as follows:
```python
def dispatch_with_args(self, body, argMap):
    'Perform callback dispatching, with enhanced arguments and recording of response'
    inputs = body.get('inputs', [])
    if isinstance(inputs, list) and len(inputs) == 1:
        if isinstance(inputs[0], list):
            inputs = inputs[0]
```

Then the above error goes away. However, the pattern matching callbacks still don't work. Am I missing something?

**Comment by GibbsConsulting** [comment]
@kino4u you're not missing anything. Pattern matching callbacks are not permitted with dpd at the moment - see #264 

Right now, your best way forward is to rewrite your app to avoid wildcard callbacks.

**Comment by GibbsConsulting** [comment]
This should now be addressed by PR #307, which has been merged and is in releases from v1.6.0 onwards.

If anything is amiss, please open a new issue.
